test/e2e/load.go

```diff
@@ -147,8 +168,41 @@ func playWithRC(c *client.Client, wg *sync.WaitGroup, ns, name string, size int)
 }
 
 func runRCLoad(c *client.Client, wg *sync.WaitGroup, ns, groupName string, size, count int) {
-	By(fmt.Sprintf("Running %v Replication Controllers with size %v and playing with them", count, size))
 	for i := 1; i <= count; i++ {
 		go playWithRC(c, wg, ns, groupName+"-"+strconv.Itoa(i), size)
 	}
 }
+
+func createRCGroup(c *client.Client, ns, groupName string, size, count, rate int) {
```

Comment:
s/rate/batchSize

I think that rate usually is something like "something per second". This parameter controls how many RC you create in one batch.